### PR TITLE
Form and card refactoring

### PR DESCRIPTION
**Pull request by MichiBerger**

### CODE REVIEW

File: src/GlobalStyles.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import { createGlobalStyle } from 'styled-components';
 
 export default createGlobalStyle`
-* {
+*,
+*::before,
+*::after {
   box-sizing: border-box;
   padding: 0;
```
Comment (Quaquaro) [inline src/GlobalStyles.js:8]: Im not sure if the padding here is a good idea.

File: src/GlobalStyles.js [hunk 2]
```diff
@@ -30,7 +32,7 @@ ul{
...
   list-style: none;
 }
-input[type="text"], textarea {
+input[type="text"], textarea, select, option {
   font-family: sans-serif;
```
Comment (Quaquaro) [inline src/GlobalStyles.js]: Use inherit here for the font-family

File: src/pages/AddPhrases.js [hunk 3]
```diff
@@ -16,7 +29,48 @@ export default function AddPhrases({ phrases, handlePhraseSubmit, showMessage, h
...
+  min-height: 250px;
+  min-width: 300px;
+  border: 1px dashed #19337a;
+  background-color: #f9f9f9;
+  border-radius: 10px;
+
+  span {
+    display: block;
+    font-size: 1.2rem;
+    color: #19337a;
```
Comment (Quaquaro) [inline src/pages/AddPhrases.js:74]: I would recommend you to define color variables

**Review by Quaquaro — APPROVED** [review]
Really good job. Lets start in the final week. :D

File: src/pages/AddPhrases.js (hunk 3 above)
Comment (MichiBerger) [reply src/pages/AddPhrases.js:74]: hey chris!thx for reviewing! that´s a good idea 👍 I´ll do it when I´ll refactor the code

**Pull request merged**